changelog/8791.doc.md

```diff
@@ -0,0 +1 @@
+The experimental feature warning for entity roles and groups has been removed from the rasa docs, and from the code where it used to appear as a print statement. 
```

Comment:
not only from the docs but also when using Rasa Open Source, right?

Reply:
Yes, @wochinge. Rasa Open Source is what I mean by:
> and from the code where it used to appear as a print statement.

Reply:
Oh true 🙈 Makes sense 👍🏻